log.c

```diff
@@ -140,7 +140,7 @@ void log_counter_info(void)
 void log_FTL_version(const bool crashreport)
 {
 	logg("FTL branch: %s", GIT_BRANCH);
-	logg("FTL version: %s", GIT_TAG);
+	logg("FTL version: %s", GIT_VERSION);
```

Comment:
Is this meant for #595?

Reply:
Yes, this got committed by accident, I'll remove it from this PR.